Makefile

```diff
@@ -371,9 +375,9 @@ helm: ## Download helm locally if necessary.
 
 .PHONY: helm-unittest-plugin
 helm-unittest-plugin: helm ## Download helm unittest plugin locally if necessary.
-	if [ -z "$(shell $(HELM) plugin list | grep unittest)" ]; then \
+	if [ -z "$(shell HOME="$(LOCALBIN)/helm-home" $(HELM) plugin list | grep unittest)" ]; then \
```

Comment:
Are we using this target `helm-unittest-plugin` and `helm-unittest`?.

Reply:
Yes, it is part of the verify target. But I guess I could try removing this change and define HOME in openshift/release instead. WDYT?